docs/backplane/requirements/acs/10-acs-managed-service.md

```diff
@@ -115,10 +163,20 @@ Permissions at cluster scope.
 * view grafanas
 * view grafanadatasources
 * view grafanadashboards
+* view grafana-admin-credentials secret
```

Comment:
Secret access is not granted pre https://github.com/openshift/managed-cluster-config/blob/master/docs/backplane/guideline.md#day-to-day